### ISSUE TITLE
Errors with topology

### ISSUE THREAD

**Issue by SBolo** [msmbuilder/msmbuilder#636]
Hi everyone,
I have some MD trajectories of a protein without the corresponding topology. I selected a relevant snapshot where the protein is folded and computed the topology from that using the pdb2gmx command of GROMACS, in particular

pdb2gmx -f snapshot.pdb -ignh

selecting AMBER99SB-ILDN as ff and TIP-3P as solvent model since those where the ones used to obtain the trajectories. Now I tried to use the ContactFeaturizer with my trajectories and topology and get the following traceback:

```
"Traceback (most recent call last):
  File "/Users/Simone/anaconda/bin/msmb", line 6, in <module>
    sys.exit(main())
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/msmbuilder-3.2.0-py2.7-macosx-10.5-x86_64.egg/msmbuilder/scripts/msmb.py", line 28, in main
    app.start()
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/msmbuilder-3.2.0-py2.7-macosx-10.5-x86_64.egg/msmbuilder/cmdline.py", line 440, in start
    instance.start()
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/msmbuilder-3.2.0-py2.7-macosx-10.5-x86_64.egg/msmbuilder/commands/featurizer.py", line 59, in start
    input_dataset = MDTrajDataset(self.trjs, topology=top, stride=self.stride, verbose=False)
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/msmbuilder-3.2.0-py2.7-macosx-10.5-x86_64.egg/msmbuilder/dataset.py", line 406, in __init__
    self._topology = _parse_topology(os.path.expanduser(topology))
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/mdtraj/core/trajectory.py", line 149, in _parse_topology
    _traj = load_frame(top, 0)
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/mdtraj/core/trajectory.py", line 291, in load_frame
    return loader(filename, frame=index, **kwargs)
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/mdtraj/formats/pdb/pdbfile.py", line 136, in load_pdb
    with PDBTrajectoryFile(filename) as f:
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/mdtraj/formats/pdb/pdbfile.py", line 232, in __init__
    self._read_models()
  File "/Users/Simone/anaconda/lib/python2.7/site-packages/mdtraj/formats/pdb/pdbfile.py", line 520, in _read_models
    self._topology.create_disulfide_bonds(self.positions[0])
IndexError: index 0 is out of bounds for axis 0 with size 0"
```

What can be the problem? Thank you very much!

**Comment by mpharrigan** [comment]
Pass the pdb file

**Comment by rmcgibbo** [comment]
I think what @mpharrigan means is that it looks like something could be wrong with the PDB file, or the loading of the PDB file. If you could post a link to the PDB file, that would help to debug the issue.

**Comment by SBolo** [comment]
Here we go:
https://drive.google.com/open?id=0B-Ewg62rFWqdM3g4MGhZUVhkeHM
It is better if I explain you exactly what I did. The .pdb file I used contains the 'HSD' residue, which has a different name in Amber-ildn ff, 'HID' (but still I'm 100% sure that the trajectories are computed with Amber). Therefore I changed 'HSD' to 'HID' manually in the .pdb file and computed the topology. I then tried to featurize the trajectories with the 'HID' topology and it wasn't working, so I tried to change 'HID' to 'HSD' back directly in the topology but it dosen't work too. Anyway, thank you for your help.
